src/openrct2/world/tile_inspector.c

```diff
@@ -389,7 +389,7 @@ sint32 tile_inspector_surface_show_park_fences(sint32 x, sint32 y, bool showFenc
 		if (!showFences)
 			surfaceelement->properties.surface.ownership &= ~0x0F;
 		else
-			update_park_fences(x << 5, y << 5);
+			update_park_fences_around_tile(x << 5, y << 5);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7704811/26784431/8d7958b6-49cb-11e7-8d62-042554b50422.png)
It looks like some calls are actually necessary. but I have removed the around-tile calls to the ones that weren't originally updating the tiles around and will push in a second